reference/3.0/Microsoft.PowerShell.Utility/Measure-Object.md

```diff
@@ -139,7 +232,7 @@ Accept wildcard characters: False
 
 ### -IgnoreWhiteSpace
 
-Ignores white space in word counts and character counts.
+Indicates that the cmdlet ignores white space in word counts and character counts.
```

Comment:
How can you ignore whitespace in words? Words are defined by whitespace. As far as I can tell this only affects character counts

```
PS>  "Four Five" | Measure-Object -Word -Line -Character

Lines Words Characters Property
----- ----- ---------- --------
    1     2          9

PS>  "Four Five" | Measure-Object -Word -Line -Character -IgnoreWhiteSpace

Lines Words Characters Property
----- ----- ---------- --------
    1     2          8
```